templates/storyboards/swift3.stencil

```diff
@@ -24,89 +20,74 @@ protocol StoryboardSceneType {
 }
 
 extension StoryboardSceneType {
-  static func storyboard() -> UIStoryboard {
-    return UIStoryboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
-  }
-
-  static func initialViewController() -> UIViewController {
-    guard let vc = storyboard().instantiateInitialViewController() else {
-      fatalError("Failed to instantiate initialViewController for \(self.storyboardName)")
-    }
-    return vc
+  static func storyboard() -> {{prefix}}Storyboard {
+    return {{prefix}}Storyboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
   }
 }
 
 extension StoryboardSceneType where Self: RawRepresentable, Self.RawValue == String {
```

Comment:
Right, just re read these UIStoryboardSegue doc, confirms that. 
Then let's keep segue as enums but move scenes as structs as you suggested!